VKapp/VKapp/Services/NetworkService/OperationQueue/ReloadTable.swift

```diff
@@ -0,0 +1,24 @@
+// ReloadTable.swift
+// Copyright © RoadMap. All rights reserved.
+
+import RealmSwift
+
+/// Describe operation save to realm after parsing
+final class ReloadTable: Operation {
+    // MARK: - Public Methods
+
+    override func main() {
+        guard let getParseData = dependencies.first as? ParseGroupData else { return }
+        let parseData = getParseData.outputData
+        do {
+            let realm = try Realm()
+            guard let oldData = RealmService.get(Group.self) else { return }
+            try realm.write {
+                realm.delete(oldData)
+                realm.add(parseData)
+            }
```

Comment:
```suggestion
  RealmService.save
```
  на 15 используешь реалм сервис, а ниже работаешь вне его